code/game/objects/items/hand_item.dm

```diff
@@ -51,7 +51,7 @@
 	item_state = "nothing"
 	force = 0
 	throwforce = 0
-	flags = DROPDEL | ABSTRACT
+	flags = DROPDEL | ABSTRACT | NODROP
```

Comment:
I mean this *works*, but it does mean that there's no way to cancel kissing someone.
Also makes the `DROPDEL` flag a bit redundant.